.travis.yml

```diff
@@ -4,4 +4,4 @@ rvm:
 before_script:
   - gem install awesome_bot
 script:
-  - awesome_bot README.md --allow-dupe
+  - awesome_bot README.md --allow-dupe --white-list webchat.oftc.net
```

Comment:
Yes I cannot solve this problem and this issue sometime happens other website. here is the link: https://github.com/dkhamsing/awesome_bot/issues/44 I would like to fix it but I am not good at ruby 😢